index.html

```diff
@@ -310,6 +319,224 @@ <h2>Contact Me</h2>
+          </div>
+        </div>
+      </div>
+      <div class="card-work3">
+        <div class="card-image3">
+          <!-- <img src="./images/d3.png" alt="card-work"> -->
+        </div>
+        <h2 class="h2">FACEBOOK 360</h2>
+      <div class="left2-block">
+          <div class="frame3">
+            <p>FACEBOOK</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1"> FULL STACK Dev</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">2015</p>
+          </div>
+          
+            <p class="primary3">
+              Exploring the future of media in Facebook's first Virtual Reality <br />app; a place to discover and enjoy 360 photos and videos on <br />Gear VR.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action2">
+            <a href="#"><p>See Project</p></a>
+          </div>
+          </div>
+      </div>  
+      <div class="card-work4">
+        <div class="card-image4">
+          <img src="./images/d5.png" alt="card-work">
+        </div>
+        <div class="left1-block">
+          <h2 class="h2">
+            Uber Navigation
+          </h2>
+          <div class="frame4">
+            <p>FACEBOOK</p>
+              <img src="./images/Counter.png" alt="">
+            <p class="p1">FULL STACK DEV</p>
+              <img src="./images/Counter.png" alt="">
+            <p class="p1">2015</p>
+          </div>
+          <p class="primary2">
+            Experimental content creation feature that allows users to<br /> add to an existing story over the course of a day without<br /> spamming their friends.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+  <div class="action1">
+  <a href="#"><p>See Project</p></a>
+  </div>
+</div>
+</div>
+</section>
+
+<section class="about1"> 
+  <div>
+<div class="top">
+  <h2 class="h2">About<br />Myself</h2>
+  <div class="primary6">
+    <p>
+      Hello I am a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, do not hesitate to contact me.
+    </p>
+  </div>
+  <div class="connect1"> 
+   <a href="#"><p>LET US CONNECT</p></a>
+  </div>
+  <div class=" social_links1">
+    <ul>
+      <li>
+        <a href="#">
+          <img class="twit1-icon" src="./images/twitter.png" alt="twit-icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="git1-icon" src="./images/github.png" alt="git-icon">
+        </a>
+      </li>
+      
+      <li>
+        <a href="#">
```

Comment:
- [ ] I would suggest you stick as much as possible with Figma design: `color`, `font-size`, `font-family`.
- [ ] The buttons of the list of technology are not the same as the Figma design. Kindly fix it.
- [ ] Your background is totally different from Figma. Kindly fix it.
- [ ] There's a `Get my Resume` button. So use it as a `button` with a border.
### your design
![his](https://user-images.githubusercontent.com/81727988/167233061-9fbbedba-5ead-440c-acb8-8595f22435a8.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/167233073-620e4cd2-3e2d-4aee-ada0-3f4bc0db6ccb.png)